Source/Core/VideoBackends/Vulkan/VKStreamBuffer.cpp

```diff
@@ -54,73 +52,50 @@ bool StreamBuffer::AllocateBuffer()
       nullptr                                // const uint32_t*        pQueueFamilyIndices
   };
 
-  VkBuffer buffer = VK_NULL_HANDLE;
-  VkResult res =
-      vkCreateBuffer(g_vulkan_context->GetDevice(), &buffer_create_info, nullptr, &buffer);
-  if (res != VK_SUCCESS)
-  {
-    LOG_VULKAN_ERROR(res, "vkCreateBuffer failed: ");
-    return false;
-  }
-
-  // Get memory requirements (types etc) for this buffer
-  VkMemoryRequirements memory_requirements;
-  vkGetBufferMemoryRequirements(g_vulkan_context->GetDevice(), buffer, &memory_requirements);
-
-  // Aim for a coherent mapping if possible.
-  u32 memory_type_index = g_vulkan_context->GetUploadMemoryType(memory_requirements.memoryTypeBits,
-                                                                &m_coherent_mapping);
+  VmaAllocationCreateInfo alloc_create_info = {};
+  alloc_create_info.flags = 0;
```

Comment:
For my implementation I used:

```
alloc_create_info.flags = VMA_ALLOCATION_CREATE_MAPPED_BIT;
alloc_create_info.usage = VMA_MEMORY_USAGE_CPU_TO_GPU;
```

Reply:
`VMA_MEMORY_USAGE_CPU_TO_GPU` is deprecated.

Reply:
Ah thanks.  I was wondering why I couldn't find anything on it.  I wonder which usage pattern is best then.